<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionSettingDialog</class>
 <widget class="QDialog" name="PrescriptionSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="generalInfoCheck">
     <property name="text">
      <string>General Infomation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="specCheck">
     <property name="text">
      <string>Specifications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="surfaceDataCheck">
     <property name="text">
      <string>Surface Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="firstOrderDataCheck">
     <property name="text">
      <string>First Order Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrescriptionSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrescriptionSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
